src/styles/blocks/card.scss

```diff
@@ -0,0 +1,100 @@
+.card {
+  width: $card-width;
+  height: 100%;
+  padding: 32px $card-padding $card-padding;
+
+  background-color: $white-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    display: block;
+    width: 160px;
+    max-height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__product-title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+
+    margin-bottom: 4px;
+
+    color: $main-color;
+  }
+
+  &__product-code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    margin-bottom: 16px;
+
+    color: $second-color;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+
+    color: $main-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $second-color;
+  }
+
+  &__price-amount {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $main-color;
+  }
+
+  &__buy-button {
+    display: block;
+    height: 40px;
+
+    background: $hover-color;
+    border: 1px solid transparent;
+    border-radius: 5px;
+
+    font-weight: 700;
+    font-size: 14px;
+    text-transform: uppercase;
+    color: $white-color;
+    text-align: center;
+    line-height: 40px;
+    text-decoration: none;
+
+    transition: color 0.3s ease-in-out,
+      background-color 0.3s ease-in-out;
```

Comment:
You can add variable for 0.3s ease-in-out if you dont need to change these values separatly